src/utils/jestMocks/base64Image.js

```diff
@@ -0,0 +1 @@
+export default 'data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIyNCIgaGVpZ2h0PSIyNCIgdmlld0JveD0iMCAwIDI0IDI0Ij48cGF0aCBkPSJNNyA3YTUgNSAwIDEgMSAxMCAwQTUgNSAwIDAgMSA3IDd6bTUgN2MtNCAwLTkgMy41LTkgNiAwIC44MjguNjU2IDIgMiAyaDE0YzEuMzI4IDAgMi0xLjE3MiAyLTIgMC0yLjUtNS02LTktNnoiLz48L3N2Zz4=';
```

Comment:
before merging, could you move this to a root-level `__mocks__/base64-image-loader.js` file, and make it a commonJS module (`module.exports =`) instead of ES6?

We're _just_ starting to create some proper "manual mocks" for testing, and [Jest expects those mocks to be in `/__mocks__`](https://facebook.github.io/jest/docs/manual-mocks.html), so most mocks will end up there.